src/server/scripts/Spells/spell_quest.cpp

```diff
@@ -1974,25 +1974,23 @@ class spell_q13264_thats_abominable : public SpellScriptLoader
             {
                 PreventHitDefaultEffect(effIndex);
 
-                if (Unit* unit = GetHitUnit())
+                if (Creature* creature = GetHitCreature())
                 {
-                    ObjectGuid objGuid = unit->GetGUID();
-
                     // Except Player
-                    if (objGuid.IsPlayer())
+                    if (creature->GetGUID().IsPlayer())
```

Comment:
a Creature* is never a Player*, their C++ class inheritance doesn't allow it. Please take the time to familiarize more with the inheritance of these classes.

Reply:
Thanks for the hints